opm/simulators/flow/SimulatorConvergenceOutput.cpp

```diff
@@ -42,14 +50,14 @@ startThread(std::string_view convOutputOptions,
     }
 
     auto convertTime = ConvergenceOutputThread::ConvertToTimeUnits {
-        [usys = eclState_.getUnits()](const double time)
+        [usys = eclState.getUnits()](const double time)
```

Comment:
copy necessary?

Reply:
> copy necessary?

We do indeed need a copy here, because the ownership of the `convertTime()` utility function is [transferred](https://github.com/OPM/opm-simulators/blob/c800c89674d996346e2066313bf3d2ff6383cca6/opm/simulators/flow/SimulatorConvergenceOutput.cpp#L54) into the convergence output thread and therefore outlives the `startThread()` function.

If you'd like me to I can write this more explicitly as
```C++
auto usysCopy = eclState.getUnits();
auto convertTime = [usys = std::move(usysCopy)](...);
```
instead.

Reply:
not at all, using capture to copy is fine, just wanted to confirm.